cbbr_build/python/geocoding.py

```diff
@@ -158,24 +164,40 @@ def geo_parser(geo):
 if __name__ == '__main__':
     # connect to postgres db
     engine = create_engine(os.environ['BUILD_ENGINE'])
+    select_query = text("SELECT * FROM _cbbr_submissions")
+    with engine.begin() as conn:
```

Comment:
We went over this yesterday but I don't think you need to do `with engine.begin() as conn:` and can just do `pd.read_sql(sql=text(select_query), con=engine.connect())`. I am happy with either so more of a FYI than anything